src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule32packagestatement/InputPackageStatement.java

```diff
@@ -2,9 +2,13 @@
         chapter3filestructure.rule32packagestatement;
 
 public class InputPackageStatement {
-  // Long line ----------------------------------------------------------------------------------------
-  // violation above 'Line is longer than 100 characters (found 103).'
+  // Long line
+  // -----------------------------------------------------------------------------------------------------
+  // violation above 'Line is longer than 100 characters (found 106).'
 
-  private int[] mInts = new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24};
-  // violation above 'Line is longer than 100 characters (found 174).'
+  private int[] mInts =
+      new int[] {
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24
+      };
```

Comment:
google formatter changed this code to be in column limit 100, this code was intentionally put here to show that column limit rule does not applies to package statements but it applies to other code.

Should I revert this and add back the violation?

Reply:
Yes, format whole file and revert certain lines to keep violation.

Reply:
>3.2 Package statement
The package statement is not line-wrapped. The column limit (Section 4.4, [Column limit: 100](https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.4-column-limit)) does not apply to package statements.

This violation is irrelevant for this chapter, we can loose it